### DISCUSSION TITLE
When using inline variable in javascript form refresh when i focus to any input fields

### DISCUSSION THREAD

**Discussion by chokies** [livewire/livewire#2653] [1. Help]
if i use @this.variable = 1; in my inline script this will refresh the dom whenever i focus to any inputs on my form.

**Comment by joshhanley** [comment]
@chokies  I've converted this to a discussion until we can narrow down what the bug is.

Can you please add some more detail to this issue?

There were questions in the issue template that we need to try and help you solve your problem, I've listed them below.

### Description

### Exact steps to reproduce

### Stripped-down, copy-pastable code snippets

**Context**
- Livewire version: [e.g. 2.2.3]
- Laravel version: [e.g. 8.3.1]
- Alpine version: [e.g. 2.3.0]
- Browser: [e.g. Chrome, Safari]
